core/src/main/scala/org/broadinstitute/dsde/rawls/entities/datarepo/DataRepoBigQuerySupport.scala

```diff
@@ -507,12 +507,12 @@ trait DataRepoBigQuerySupport extends LazyLogging {
         // the given fromTable may be either the from or the to in the relationship model
         // depending on the direction the relationship is being traversed
         val entityJoin = fromTable.name match {
-          case forward if forward.equalsIgnoreCase(relationshipModel.getFrom.getTable) =>
+          case forward if forward.equals(relationshipModel.getFrom.getTable) =>
             val fromColumn = EntityColumn(snapshotModel, fromTable, relationshipModel.getFrom.getColumn)
```

Comment:
Do we want to be case insensitive on expressions? Note to self, this needs to be tested